### PR TITLE
Akka integration

### PR DESCRIPTION
**Pull request by chef-plankton**
[ ] Before opening a pull request, please read the [contributing guidelines](https://github.com/pancakeswap/pancake-frontend/blob/master/CONTRIBUTING.md) first
[ ] If your PR is work in progress, open it as `draft`
[ ] Before requesting a review, all the checks need to pass
[ ] Explain what your PR does

### CODE REVIEW

File: apps/web/src/views/Swap/index.tsx [hunk 1]
```diff
@@ -115,11 +115,22 @@ export default function Swap() {
...
+      toggleSetAkkaActiveToTrue();
+    }
+  }, [independentField])
+  
   // Check if pancakeswap route is better than akka route or not
   useEffect(() => {
     if (akkaRouterTrade?.route?.returnAmountWei && v2Trade?.outputAmount) {
       if (v2Trade?.outputAmount.greaterThan(JSBI.BigInt(akkaRouterTrade?.route?.returnAmountWei))) {
-        toggleSetAkkaModeToFalse()
+        toggleSetAkkaModeToTrue()
```
Comment (MasterGordon) [inline apps/web/src/views/Swap/index.tsx]: Is this change intentional calling toggleSetAkkaModeToTrue in both cases?

Comment (chef-plankton) [reply apps/web/src/views/Swap/index.tsx]: just to test AKKA backend ;)

Comment (chef-plankton) [reply apps/web/src/views/Swap/index.tsx]: check the next commit "add new contracts for akka"
